dtfs-central-api/src/v1/controllers/tfm/facility/tfm-get-facilities.controller.js

```diff
@@ -4,10 +4,10 @@ const db = require('../../../../drivers/db-client');
 const findFacilitiesByDealId = async (dealId) => {
   const collection = await db.getCollection('tfm-facilities');
 
-  // NOTE: only GEF facilities have applicationId.
+  // NOTE: only GEF facilities have dealId.
   // this could be adapted so that we get the deal, check dealType,
-  // then search for either dealId or applicationId.
-  const facilities = await collection.find({ 'facilitySnapshot.applicationId': { $eq: ObjectID(dealId) } }).toArray();
+  // then search for either dealId or dealId.
```

Comment:
Might be able to do this improvement listed in comment now - not required for this PR :)